modules/standard/Memory/Diagnostics.chpl

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright 2020-2021 Hewlett Packard Enterprise Development LP
+ * Copyright 2004-2019 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ * 
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * 
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+  The :mod:`Diagnostics` module provides procedures which report information
+  about memory usage.  With one exception, to use these procedures you
+  must enable memory tracking.  Do this by setting one or more of the
+  config vars below, using appropriate ``--configVarName=value`` or
+  ``-sconfigVarName=value`` command line options when you run the
+  program.  If memory tracking is not enabled, calling any procedure
+  described here, other than :proc:`locale.physicalMemory`, will cause
+  the program to halt with an error message.
+
+  ``memTrack``: `bool`:
```

Comment:
The output for these guys looked a little weird to me and I realized it was because they are not actually defined in the module.  It might be good to more closely mimic how the generated documentation would look if they were actually config vars or config params - doing so would allow us to link to them in the future if we so chose.